src/components/NotFound.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+
+const NotFound = () => {
+    return <NoContent />;
+  };
+  
+const NoContent = () => {
+    const data = 'No data found!';
```

Comment:
If it is not dynamic data I would suggest to move it out of the scope of component

const data = ''

const NoContent = () => <h1>{ data }</h1>

or even better to pass it as a prop
const NoContent = ({ data }) => <h1>{ data }</h1>
<NoContent data={ data }>

Reply:
Updated